client-runtime/src/main/java/com/microsoft/rest/http/ContentType.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.rest.http;
+
+/**
+ * The different values that we commonly use for Content-Type headers.
+ */
+public abstract class ContentType {
```

Comment:
If this class is intended to not be instantiated or subclassed I'd prefer it be final with a private empty constructor. It might be nice to make it an enum if we can swing it at all the usage sites. That might not be very straightforward though.

Reply:
I agree about making it final with a private constructor. I went back and forth on making this an enum or just leaving it as a collection of constant strings. Let's leave it as strings for now with the option to make it an enum later.

Reply:
Done.